app/main.py

```diff
@@ -2,0 +1,12 @@
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[0] != "cp":
```

Comment:
The error message 'You line should start with cp' is not very clear. It would be better to say 'The command should start with 'cp''.